fe/fe-core/src/test/java/com/starrocks/catalog/DropPartitionTest.java

```diff
@@ -132,7 +132,7 @@ public RunMode getCurrentRunMode() {
 
         new MockUp<SharedNothingStorageVolumeMgr>() {
             @Mock
-            public StorageVolume getStorageVolume(String svKey) throws AnalysisException {
+            public StorageVolume getStorageVolumeByName(String svKey) throws AnalysisException {
```

Comment:
change the parameter name accordingly, avoid confusion.

Reply:
The name of parameter should be `svKey`, because it is a shared nothing storage volume manager. And I have changed all of the other functions like this.

Reply:
I am a little confused, should the concept consistent no matter it is a shared nothing mode or shared data mode. so what's the following three concepts mapping to filestore?

* volume id -> uuid
* svKey -> ??
* svName -> ??

Reply:
volume id -> uuid
svKey -> fsKey
svName -> fsName